src/main/java/io/leangen/graphql/GraphQLSchemaGenerator.java

```diff
@@ -142,14 +142,30 @@
     private final RelayMappingConfig relayMappingConfig = new RelayMappingConfig();
     private final Set<GraphQLType> additionalTypes = new HashSet<>();
 
-    private static final String QUERY_ROOT = "QUERY_ROOT";
-    private static final String MUTATION_ROOT = "MUTATION_ROOT";
-    private static final String SUBSCRIPTION_ROOT = "SUBSCRIPTION_ROOT";
+    private final String queryRoot;
+    private final String mutationRoot;
+    private final String subscriptionRoot;
 
     /**
      * Default constructor
      */
-    public GraphQLSchemaGenerator() {}
+    public GraphQLSchemaGenerator() {
+        this.queryRoot = "QUERY_ROOT";
```

Comment:
Maybe make _Query_, _Mutation_ and _Subscription_ the default names? Those seem to be the most often chosen ones.

Reply:
I was thinking that upper-case names are used intentionally to prevent potential name conflicts.
I personally would prefer to have pascal-case defaults so do you think I can change it?

Reply:
Changed, let's see if it causes any problems.

Reply:
The original choice was indeed to avoid clashes... But all the examples on the web use these defaults, I figure the users sort of expect them. Let's see. If I get complaints, I'll reevaluate the choice.
By pascal-case defaults, you meant `QueryRoot` etc or just `Query` like now?

Reply:
By pascal-case I meant both `QueryRoot` and `Query`.
I used usual names (Query/Mutation/Subscription) of course.